onadata/libs/serializers/monthly_submissions_serializer.py

```diff
@@ -0,0 +1,35 @@
+"""
+Monthly submissions serializer
+"""
+from rest_framework import serializers
+
+
+class MonthlySubmissionsListSerializer(serializers.ListSerializer):
+
+    def to_representation(self, data):
+        result = super(MonthlySubmissionsListSerializer,
+                       self).to_representation(data)
+        states = [True, False]
+        result_dictionary = {}
+
+        def get_result():
+            for state in states:
```

Comment:
I think you should be able to do this without `states` and the extra outer loop around `states`